var/spack/repos/builtin/packages/py-quart/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyQuart(PythonPackage):
+    """A Python ASGI web microframework with the same API as
+    Flask."""
+
+    homepage = "https://gitlab.com/pgjones/quart/"
+    pypi     = "Quart/Quart-0.16.3.tar.gz"
+
+    version('0.16.3', sha256='16521d8cf062461b158433d820fff509f98fb997ae6c28740eda061d9cba7d5e')
+
+    depends_on('python@3.7:',           type=('build', 'run'))
+    depends_on('py-setuptools',         type='build')
```

Comment:
Setuptools is only needed if poetry isn't used as the build system, you can remove this